cosmo_tester/framework/cluster.py

```diff
@@ -213,9 +213,13 @@ def verify_services_are_running(self):
         status = self.client.manager.get_status()
         for service in status['services']:
             for instance in service['instances']:
-                assert instance['SubState'] == 'running', \
-                    'service {0} is in {1} state'.format(
-                            service['display_name'], instance['SubState'])
+                if (
+                    service['display_name'] == 'Cloudify Stage'
```

Comment:
That's only true if the ID also remains the same- it feels likely that if we change the display name we'll also change the ID, because it'll probably be the case that we're not calling it 'Stage' any more. It'd be nice if the status didn't display premium services on community managers... worth a Jira?